shared/actions/tracker.js

```diff
@@ -346,7 +300,7 @@ function updateUserInfo (userCard: UserCard, username: string, getState: () => {
         followingCount: userCard.following,
         followsYou: userCard.theyFollowYou,
         bio: userCard.bio,
-        avatar: `${serverURI}/${username}/picture`,
+        avatar: `https://keybase.io/${username}/picture`,
```

Comment:
we switched serverURI to api.keybase.io which doesn't have the right certs we need so we see warnings in the console now. this fixes that